GOVERNANCE.rst

```diff
@@ -0,0 +1,183 @@
+
+In particular, we will outline
+
+* how we expect all members of the sktime community to behave,
+* what priviledges and responsibilities authors and maintainers have,
+* how we curate and select new contributions and existing functionality,
+* how we make decisions,
+* how we acknowledge contributions.
+
+
+Code of Conduct
+---------------
+We value the participation of every member of our community and want to ensure an that every contributor has an enjoyable and fulfilling experience. Accordingly, everyone who participates in the sktime project is expected to show respect and courtesy to other community members at all times.
+
+To make clear what is expected, we ask all members of the community to
+conform to our `Code of Conduct <https://github
+.com/alan-turing-institute/sktime/blob/master/CODE_OF_CONDUCT.rst>`_.
+
+Authorship and maintainership of algorithms
+-------------------------------------------
+
+Authors and maintainers of algorithms have special rights and duties with regard to changes and maintenance of their algorithms and associated documentation, including doc strings and tutorial notebooks.
+
+Algorithms are encapsulated in classes. To faciliate authorship and maintainership questions, each algorithm is written in a separate file.
+
+The author is the contributor who implements and contributes the algorithm to the project. The maintainer is the contributor who is responsible for maintaining an algorithm and the first point of contact for users and other contributors for all issues, questions, and proposals regarding their algorithm. The author automatically becomes the first maintainer of the algorithm.
+
+* **Veto right.** The maintainer has the right to veto any proposed changes to their algorithm within a time frame of 1 week. This does not extend to change to the common framework and API.
+
+* **Maintenance duty.** The maintainer is responsible for maintaining an algorithm. This includes understandable and instructive documentation, bugs fixes, unit testing, coding quality and style improvements, and compliance with sktime's common API. Maintainers should reply to a new issue on their algorithm within 1 week, and demonstrate a significant effort to fix any issue within 1 month.
+
+Any maintainer is expected to give up their role if they can no longer fulfil their responsibilities. If a  maintainer fails to fulfil their responsibilities in given time frame, the maintainer loses their role and all rights that come with it.
+
```

Comment:
This can be seen both from the pragmatic side (keeping things running) but can also appear as punishment.
To satisfy the governance principle of "regulatory certainty", for every restrition of rights, it needs to be very clearly said what it means what the conditions are, who ultimately decides, what the appeals process is.
E.g., what does "failing to fulfil their responsibilities" mean, and who decides? What is the nature of appeals?

E.g., maintainer disappears and it turns our 3 months later they were in hospital. But maintainership was transferred after 1 month since for all practical purposes it looked like they were gone. What then?

Reply:
Not sure, any ideas? We could reserve the right of the technical committee to revise any conflict/decision.

Reply:
see above - does that make sense?